src/components/EventStatus.js

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import { Alert } from './publiq-ui/Alert';
+import { Button, ButtonVariants } from './publiq-ui/Button';
+import { Page } from './publiq-ui/Page';
+import { Spinner } from './publiq-ui/Spinner';
+
+const EventStatus = ({ offer, loading }) => {
```

Comment:
OfferStatus would actually be more accurate right? But yeah for consistency's sake in frontend/backend I'll keep it named like this.

Reply:
It's not called `eventStatus` in the backend btw. We named it simply `status` for this reason. But Corneel keeps referring to it as "eventStatus" 😅 

So yes you can call this `offerStatus` (or just `status`), that would be much clearer I think!

Reply:
done 👍🏻